connect-inject/handler.go

```diff
@@ -8,26 +8,21 @@ import (
 	"net/http"
 	"strconv"
 
-	"github.com/deckarep/golang-set"
+	mapset "github.com/deckarep/golang-set"
 	"github.com/go-logr/logr"
```

Comment:
Was this added due to linter or IDE? (My IDE automatically adds it still so I can't quite tell but if the linter wants this that's great!